File: Python3/1832_check_if_the_sentence_is_pangram.py
# https://leetcode.cn/problems/check-if-the-sentence-is-pangram/

class Solution:
    def checkIfPangram(self, sentence):
        """
        Inputs:
            sentence: str
        Outputs:
            res: bool
        """
        cnt = [0 for _ in range(26)]
        for ch in sentence:
            cnt[ord(ch) - ord("a")] = 1
        res = True
        for i in cnt:
            if i != 1:
                res = False
                break
        return res

File: Cpp/1832_check_if_the_sentence_is_pangram.cpp
// https://leetcode.cn/problems/check-if-the-sentence-is-pangram/

class Solution {
public:
    bool checkIfPangram(string sentence) {
        vector<int> cnt(26,0);
        for (char ch: sentence) {
            cnt[ch - 'a'] = 1;
        }
        bool res = true;
        for (int i: cnt) {
            if (i != 1) {
                res = false;
                break;
            }
        }
        return res;
    }
};
